examples/vue.test.js

```diff
@@ -2600,6 +2600,40 @@ describe('Vue', function() {
       // acquit:ignore:end
     });
   });
+  it('vue-websocket', async function() {
+    // acquit:ignore:start
+    this.timeout(10000);
+    const browser = await puppeteer.launch({ headless: false });
+    const page = await browser.newPage();
+    const fn = function() {
+    // acquit:ignore:end
+      const app = new Vue({
+        data: () => ({ time: null }),
+        errorCaptured: function(err) {
```

Comment:
[Vue's errorCaptured hook](https://masteringjs.io/tutorials/vue/error-handling) only catches errors that are bubbled up from child components. Since there's no child components, this `errorCaptured` hook will never catch any errors, so best to get rid of it.